testsuite/docs-examples-python/run.py

```diff
@@ -6,7 +6,7 @@
 
 
 # Prep:
-command += run_app("cmake -E copy ../common/grid-small.exr grid.exr")
+command += run_app("cmake -E copy ${OIIO_TESTSUITE_ROOT}/common/grid-small.exr grid.exr")
```

Comment:
Did something not work before?

I'm not sure this is what you want here. The test will be run from within `build/testsuite/<testname>`, and it really is ../common where the grid-small.exr is located.

I believe OIIO_TESTSUITE_ROOT is the *source* testsuite, not the copy in build, if you know what I mean.

Reply:
It was because the relative path resolution was failing when running "ctest" as follows:
```console
ctest --test-dir ./build 
```

I just now realized you provided a way to run single tests with `make test TEST=...`, I thought using an absolute path would offer more flexibility. Also, being in "source" is not necessarily a blocker, as the copied image will be there:
https://github.com/buddly27/OpenImageIO/blob/pr3992/testsuite/common/grid-small.exr

We could also use a CMAKE variable (e.g. [CMAKE_CURRENT_SOURCE_DIR](https://cmake.org/cmake/help/latest/variable/CMAKE_CURRENT_SOURCE_DIR.html)) but I'm not entirely sure if it would be resolved as expected within the python scripts.

Reply:
The case I'm actually concerned about is that sometimes (willing to entertain the notion that it's nonstandard and not necessarily a good idea) I do this to run a test by hand, completely outside of ctest/cmake/make, usually when I'm tracking down some kind of broken behavior:

    cd build/testsuite/foo
    python3 ../runtest.py

But I'm not overly worried about this. I can probably find another way to do the kind of debugging I do this way.

Reply:
But my point is, if you execute the test script by hand outside of ctest, the OIIO_TESTSUITE_ROOT environment variable will not be set!

Reply:
Oh, I see, this will be a problem indeed. Then what about using the "test_source_dir" variable that is defined within `runtest.py`?
```python
command += run_app("cmake -E copy " + test_source_dir + "/../common/grid-small.exr grid.exr")
```

Reply:
sure, I think that's fine

Reply:
Updated!